dds/DCPS/ValueWriter.h

```diff
@@ -24,6 +25,47 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace DCPS {
 
+struct VWriterMemberParam {
```

Comment:
I haven't thought through how the value reader will look like yet. I think it can skip `must_understand` and `optional` because the caller probably has it already from the type info.

Reply:
I lean toward `MemberParam`.